src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java

```diff
@@ -67,13 +66,12 @@ public String getBaseURL() {
         return this.baseURL;
     }
 
-    public CodeCoverageMetrics getParentCoverage(Differential differential) {
```

Comment:
minor refactor to remove dependency on Differential class when all we need is the SHA of the parent commit